### ISSUE TITLE
Integrate Beyond 20 buttons on sites other than DNDBeyond?

### ISSUE THREAD

**Issue by gvorbeck** [kakaroto/Beyond20#976]
**Is your feature request related to a problem? Please describe.**
I'm building a D&D tool in react that keeps track of a lot of the same information that DNDBeyond does. I'd love it if my dice-rolling mechanism could be picked up by Beyond 20 and sent to FVTT.

**Describe the solution you'd like**
I've looked for an API or open issue that is similar, but don't seem to see any.

Right now, I have a React component called Dice that users can use in their notes like so: `A goblin horde of <Dice r="2d10-2" /> goblins attacks!` that creates a button that would roll those dice when clicked on and gives the result on screen. I would love to be able to have that button also alert Beyond 20 to those results and send it to FVTT.

**Additional context**
<img width="1206" alt="Screen Shot 2022-04-13 at 4 44 40 PM" src="https://user-images.githubusercontent.com/1986380/163267327-3ff177f7-53d9-4ef2-ac09-36da49e9aaaf.png">

**Comment by Aeristoka** [comment]
I'm fairly certain this is going to fall under this category: https://beyond20.here-for-more.info/faq#can-you-add-support-for-insert-new-vtt-name

TL;DR: Not gonna happen.

I'll leave it to @kakaroto to comment further

**Comment by gvorbeck** [comment]
Aw man. @kakaroto feel free to close if this isn't something you plan on working on. Love your extension otherwise :D

**Comment by kakaroto** [comment]
> Aw man. @kakaroto feel free to close if this isn't something you plan on working on. Love your extension otherwise :D

Sorry, that's indeed not something I plan on adding support for. Good luck with your tool though, and feel free to make your own extension from forking Beyond20. As long as the new extensions respects the license of Beyond20, I have no issues with someone forking it for their own site.
Thanks!
